src/components/Sponsor/Section/Sponsors.tsx

```diff
@@ -11,26 +17,40 @@ const SponsorsSection = () => {
   const [sponsors, setSponsors] = useState([]);
   const { loadData } = useContext(CMSContext);
 
+  const tierSponsors: { [key: string]: ISponsor[] } = {};
+
+  sponsors.forEach((sponsor: { fields: ISponsor }) => {
+    if (!tierSponsors[sponsor.fields.tier]) {
+      tierSponsors[sponsor.fields.tier] = [];
+    }
+    tierSponsors[sponsor.fields.tier].push(sponsor.fields);
+  });
+
+  console.log(tierSponsors);
+
   useEffect(() => {
     loadData("sponsors", setSponsors);
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
   return (
     <Wrapper id="sponsors">
-      <SectionTitle variation="white">Parceiros</SectionTitle>
-      <SponsorsContainer>
-        {sponsors
-          .sort(
-            (
-              { fields: a }: { fields: ISponsor },
-              { fields: b }: { fields: ISponsor },
-            ) => a.name.localeCompare(b.name),
-          )
-          .map((sponsor: { fields: ISponsor }, index) => (
-            <Sponsor key={index} data={sponsor.fields} />
-          ))}
-      </SponsorsContainer>
+      <SectionTitle>Parceiros</SectionTitle>
+      <Description>
+        Os seguintes parceiros contribuem para o desenvolvimento do projeto.
+      </Description>
```

Comment:
The gap here is 1.5rem, it's best if just between this two the gap  is 0.75rem, giving a more unity feeling. The other gaps in the section are fine